src/pair_hybrid.h

```diff
@@ -33,6 +33,7 @@ class PairHybrid : public Pair {
   friend class Respa;
   friend class Info;
   friend class PairDeprecated;
+  friend class ComputePressure;
```

Comment:
not clear why this is needed, no_virial_fdotr_compute is a public va
raible? may be moot, if my other suggestion is followed

Reply:
This is a leftover from the original implementation, that didn't use force->pair_match() but would access the `PairHybrid::styles` array, which is protected. It can indeed be removed. Also, the `nsub` class variable is no longer needed because of that change.